tests/data/scripts_pythondistdeps/test-requires.yaml

```diff
@@ -0,0 +1,93 @@
+setuptools:
+    wheel:
+        '41.6.0': ['2.7', '3.7', '3.9']
+    sdist:
+        '41.6.0': ['2.7', '3.7', '3.9', '3.10']
+pip:
+    wheel:
+        '19.1.1': ['2.7', '3.7']
+        '20.0.2': ['3.9']
+    sdist:
+        '20.0.2': ['3.11']
+packaging:
+    wheel:
+        '19.0': ['2.7', '3.7']
+        '20.1': ['3.9']
+attrs:
+    sdist:
+        '19.1.0': ['2.7', '3.9']
+pyparsing:
+    wheel:
+        '2.4.0': ['2.7', '3.7', '3.9']
+six:
+    wheel:
+        '1.12.0': ['2.7', '3.7', '3.9']
+tox:
+    wheel:
+        '3.13.2': ['2.7', '3.7', '3.9']
```

Comment:
tox 3.14 has:

```
install_requires =
    importlib-metadata >= 0.12, <1;python_version<"3.8"
```

I would like to see it tested: that importlib-metadata is required on 3.7 but not 3.8. Unfortunately 3.13.2 doesn't yet do this.

Or is there some other package here that does this?

Reply:
I've switched to `tox 3.14`, and also `kubernetes` and `astroids` have these conditionals. I checked the outputs, they look good.